packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -859,6 +859,17 @@ realm_sync_session_progress_notifier_callback(void *userdata, uint64_t transferr
     jni_check_exception(env);
 }
 
+void
+realm_sync_session_connection_state_change_callback(void *userdata, realm_sync_connection_state_e old_state, realm_sync_connection_state_e new_state) {
+    auto env = get_env(true);
+
+    static JavaMethod java_callback_method(env, JavaClassGlobalDef::connection_state_change_callback(), "onChange", "(II)V");
```

Comment:
This needs to be added to the proguard file?

Reply:
Yep. I actually left it out intentionally to see if CI was catching it now ... and it did. Have added it now.